addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17580,4 +17580,15 @@ msgstr ""
 #: system/settings/settings.xml
 msgctxt "#38017"
 msgid "Region A - Americas, East Asia and Southeast Asia. Region B - Africa, Middle East, Southwest Asia, Europe, Australia, New Zealand. Region C - Central Asia, mainland China, Mongolia, South Asia, Belarus, Russia, Ukraine, Kazakhstan."
+
+#. Setting #38118 "Settings -> System -> Audio output -> Support 8 channel DTS-HD audio decoding"
+#: system/settings/settings.xml
+msgctxt "#38118"
+msgid "Support 8 channel DTS-HD audio decoding"
+msgstr ""
+
+#. Description of setting #38118 Settings -> System -> Audio output -> Support 8 channel DTS-HD audio decoding
+#: system/settings/settings.xml
+msgctxt "#38119"
+msgid "Enables decoding of high quality DTS-HD audio streams. Note: This increases CPU load and is only available when DTS and DTS-HD audio passthrough are disabled."
```

Comment:
Not really. Remember passthrough is off by default.
Am 05.09.2015 2:53 nachm. schrieb "uNiversaI" <notifications@github.com>:

> In addons/resource.language.en_gb/resources/strings.po
> <https://github.com/xbmc/xbmc/pull/7102#discussion_r38810503>:
>
> > @@ -17580,4 +17580,15 @@ msgstr ""
> >  #: system/settings/settings.xml
> >  msgctxt "#38017"
> >  msgid "Region A - Americas, East Asia and Southeast Asia. Region B - Africa, Middle East, Southwest Asia, Europe, Australia, New Zealand. Region C - Central Asia, mainland China, Mongolia, South Asia, Belarus, Russia, Ukraine, Kazakhstan."
> > +
> > +#. Setting #38118 "Settings -> System -> Audio output -> Support 8 channel DTS-HD audio decoding"
> > +#: system/settings/settings.xml
> > +msgctxt "#38118"
> > +msgid "Support 8 channel DTS-HD audio decoding"
> > +msgstr ""
> > +
> > +#. Description of setting #38118 Settings -> System -> Audio output -> Support 8 channel DTS-HD audio decoding
> > +#: system/settings/settings.xml
> > +msgctxt "#38119"
> > +msgid "Enables decoding of high quality DTS-HD audio streams. Note: This increases CPU load and is only available when DTS and DTS-HD audio passthrough are disabled."
>
> You never see this message until DTS and DTS-HD audio passthrough are
> disabled.
>
> So perhaps the DTS and DTS-HD audio passthrough should actually include a
> mention that in order for this option to work you must disable them else
> the above message is pointless.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/7102/files#r38810503>.
>